twisted/scripts/_twistd_unix.py

```diff
@@ -198,6 +200,38 @@ def preApplication(self):
         self.oldstderr = sys.stderr
 
 
+    def _formatException(self, exception):
```

Comment:
minor comment
I think that this should have a less generic name :) so that people will not be tempted to use it for formatting any exception :)

also, if this fixed a problem in Python3 I think that this should refect in the releas notes and the Trac ticket description / title